cmd/promtool/main_test.go

```diff
@@ -23,6 +23,21 @@ import (
 	"github.com/prometheus/prometheus/util/testutil"
 )
 
+func TestAddScheme(t *testing.T) {
+	url := "google.com"
+	urlWithScheme, err := addScheme(url)
+	testutil.Ok(t, err)
+	if urlWithScheme != "http://google.com" {
+		t.Errorf("unexpected value %s for urlWithScheme", urlWithScheme)
+	}
+	url = "http://prometheus.io"
+	urlWithScheme, err = addScheme(url)
+	testutil.Ok(t, err)
+	if urlWithScheme != "http://prometheus.io" {
+		t.Errorf("unexpected value %s for urlWithScheme", urlWithScheme)
+	}
+}
```

Comment:
this is a lot of repetition, can you make the test cases a []map[string]string?

e.g.

```
for input, result := range map[string]string{
 "google.com": "http://google.com",
} {
   ...
}
```

thanks

Reply:
Not that we also have testutl.Equals.

Reply:
Thanks for pointing this out. I've fixed this in the latest commit.